docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: model
+title: Sentence Entity Resolver for SNOMED codes (procedures and measurements)
+author: John Snow Labs
+name: sbiobertresolve_clinical_snomed_procedures_measurements
+date: 2021-11-15
+tags: [en, licensed, clinical, entity_resolution]
+task: Entity Resolution
+language: en
+edition: Spark NLP for Healthcare 3.3.2
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps medical entities to SNOMED codes using `sent_biobert_clinical_base_cased` Sentence Bert Embeddings. The corpus of this model includes `Procedures` and `Measurement` domains.
+
+## Predicted Entities
+
+
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/24.Improved_Entity_Resolvers_in_SparkNLP_with_sBert.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_clinical_snomed_procedures_measurements_en_3.3.2_3.0_1636985738813.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+documentAssembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("ner_chunk")
+
+sbert_embedder = BertSentenceEmbeddings.pretrained("sent_biobert_clinical_base_cased", "en")\
+      .setInputCols(["ner_chunk"])\
+      .setOutputCol("sbert_embeddings")
+
+resolver = SentenceEntityResolverModel.pretrained("sbiobertresolve_clinical_snomed_procedures_measurements", "en", "clinical/models") \
+      .setInputCols(["ner_chunk", "sbert_embeddings"]) \
+      .setOutputCol("cpt_code")
+
+pipelineModel = PipelineModel(
+    stages = [
+        documentAssembler,
+        sbert_embedder,
+        resolver])
+
+l_model = LightPipeline(pipelineModel)
+result = l_model.fullAnnotate(['coronary calcium score', 'heart surgery', 'ct scan', 'bp value'])
+
+```
+```scala
+val document_assembler = DocumentAssembler()\
+  .setInputCol("text")\
+  .setOutputCol("ner_chunk")
+
+val sbert_embedder = BertSentenceEmbeddings.pretrained("sent_biobert_clinical_base_cased", "en")\
+     .setInputCols(["ner_chunk"])\
+     .setOutputCol("sbert_embeddings")
+
+val resolver = SentenceEntityResolverModel.pretrained("sbiobertresolve_clinical_snomed_procedures_measurements", "en", "clinical/models) \
+     .setInputCols(["ner_chunk", "sbert_embeddings"]) \
+     .setOutputCol("cpt_code")
+
+val pipelineModel= new PipelineModel().setStages(Array(document_assembler, sbert_embedder, resolver))
+val l_model = LightPipeline(pipelineModel)
```

Comment:
Switch from LightPipeline inference to standard inference

Reply:
@josejuanmartinez 
We in many cases use `LightPipeline` in our demo/model pages and we suggest this to our customers too. Why do you want us to change it? Is there a specific purpose, I didn't get it 😄